redaxo/src/addons/cronjob/lib/cronjob.php

```diff
@@ -14,6 +14,9 @@ abstract class rex_cronjob
     /** @var string */
     private $message = '';
 
+    /**
+     * @param string $class
+     */
     final public static function factory($class)
```

Comment:
Da die Methode final ist, könnte man hier zum Beispiel auch gut einen echten Type hint stattdessen hinzufügen.

Reply:
Solche generellen dinge würde ich dann in einem rutsch über die ganze codebase machen.. jetzt erstmal phpdoc only